src/System.Management.Automation/engine/DataStoreAdapterProvider.cs

```diff
@@ -251,6 +251,14 @@ public override string ToString()
         /// </summary>
         public bool VolumeSeparatedByColon { get; internal set; } = true;
 
+        /// <summary>
+        /// Gets the path separators characters for this provider.
+        /// </summary>
+        /// <returns>
+        /// Array of strings representing the path separators characters for this provider.
+        /// </returns>
+        public ReadOnlyCollection<string> PathSeparator { get; internal set; } = new ReadOnlyCollection<string>(new List<string>());
```

Comment:
Just wondering, should we allow string that is more than 1 chars to be used as the path separator? Maybe `ReadOnlyCollection<char>`?

Reply:
Also, as for the default value of this property, maybe it should be `null` by default?

Reply:
Do you want me to apply these changes?

Reply:
I agree with the changes.

Reply:
I want to be discreet here as it's a public API.
@SteveL-MSFT @joeyaiello what do you think?

Reply:
@daxian-dbw's feedback makes sense to me.  I'm not aware of any situation where the separator is more than a single character.